src/components/CreateEventForm/CreateEventForm.jsx

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import {
+  FormLabel,
+  Input,
+  Select,
+  FormControl,
+  FormErrorMessage,
+  Button,
+  Textarea,
+} from '@chakra-ui/react';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useForm } from 'react-hook-form';
+import * as yup from 'yup';
+import styles from './CreateEventForm.module.css';
+
+
+const schema = yup.object({
+    id: yup.string().required('ID required').max(10, "ID exceeds 10 character limit"),
+    host: yup.string().required('Host required').max(50, "Host exceeds 50 character limit"),
+    title: yup.string().required('Title required').max(50, "Title exceeds 50 character limit"),
+    event_type: yup.string().required('Email required'),
+    subject: yup.string().required('Subject required'),
+    description: yup.string().required('Description required').max(50, "Description exceeds 50 character limit"),
+    year: yup.string().required('Year required'),
+});
+
+
+function CreateEventForm() {
+    const {
+      register,
+    //   control,
+      handleSubmit,
+      formState: { errors },
+    } = useForm({
+      resolver: yupResolver(schema),
+    });
+
+
+    return (
+      <div className={styles['form-padding']}>
```

Comment:
pls use change div to chakra Box and if the css is very minimal and only used once, use chakra's in-line styling